src/FEValues/face_integrals.jl

```diff
@@ -6,6 +6,14 @@ cell's face.
 """
 face_to_element_transformation
 
+"""
+    element_to_face_transformation(point::AbstractVector, cell::AbstractCell{AbstractRefShape}, face::Int)
+
+Transform quadrature point from ND coordinates on the cell's face to face's reference
+(N-1)D coordinates.
```

Comment:
`ND` and `(N-1)D` might be confusing to read. Maybe we can just write it out to N dimensional and N-1 dimensional. Not sure what a good way is.